doc/geany.txt

```diff
@@ -2601,6 +2601,10 @@ gio_unsafe_save_backup            Make a backup when using GIO unsafe file     f
 keep_edit_history_on_reload       Whether to maintain the edit history when    true        immediately
                                   reloading a file, and allow the operation
                                   to be reverted.
+reload_clean_doc_on_file_change   Whether to automatically reload documents    false       immediately
+                                  which have changed on disk.
```

Comment:
Probably should say "...documents that have no changes but which have changed on disk...".  It should be spelt out because I don't think the term "clean" is used anywhere else, I think we only use "unchanged".